requirements.txt

```diff
@@ -14,4 +14,8 @@
 # Example:
 # numpy==1.13.3
 # scipy==1.0
-
+pytest==4.3.0
```

Comment:
Dependencies should be added to setup.cfg. See the documentation [here](https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files).